barretenberg/cpp/src/barretenberg/plonk_honk_shared/execution_trace/mega_execution_trace.hpp

```diff
@@ -294,7 +294,7 @@ static constexpr TraceStructure CLIENT_IVC_BENCH_STRUCTURE{ .ecc_op = 1 << 10,
                                                             .elliptic = 9000,
                                                             .aux = 136000,
                                                             .poseidon2_external = 2500,
-                                                            .poseidon2_internal = 14000,
+                                                            .poseidon2_internal = 14500,
```

Comment:
This was very close and crossed the line due to one new field element in the mega proof. Bumped it with some extra room